mlir-tensorrt/test/python/mlir_tensorrt_compiler/compiler_api/test_options_context.py

```diff
@@ -27,4 +27,4 @@
 
 
 # CHECK: InvalidArgument: InvalidArgument: non-existent-options-type is not a valid option type. Valid options were: stable-hlo-to-executable
-# CHECK: --tensorrt-timing-cache-path= --device-infer-from-host=true --debug-only= --executor-index-bitwidth=64 --entrypoint=main --plan-clustering-disallow-host-tensors-in-tensorrt-clusters=false --tensorrt-workspace-memory-pool-limit=1073741824 --device-max-registers-per-block=65536 --tensorrt-strongly-typed=false --tensorrt-layer-info-dir= --device-compute-capability=86 --debug=false --mlir-print-ir-tree-dir= --disable-tensorrt-extension=false --tensorrt-builder-opt-level=3 --tensorrt-engines-dir=
+# CHECK: Options[--tensorrt-timing-cache-path= --device-infer-from-host=true --debug-only= --executor-index-bitwidth=64 --entrypoint=main --device-max-registers-per-block=65536 --tensorrt-workspace-memory-pool-limit=1073741824 --plan-clustering-disallow-host-tensors-in-tensorrt-clusters=false --tensorrt-strongly-typed=false --device-compute-capability=86 --tensorrt-layer-info-dir= --debug=false --mlir-print-ir-tree-dir= --disable-tensorrt-extension=false --tensorrt-engines-dir= --tensorrt-builder-opt-level=3 --device-max-shared-memory-per-block-kb=48]
```

Comment:
But we do want at least a sanity check that the options are being printed correctly right? Maybe I can limit it to just a few options that we can explicitly set?

Reply:
ok, yeah, I might have been too heavy handed there. I'd say just checking 1-2 items is best, otherwise you'll need to update it every time we update the options